plugins/pulp_rpm/plugins/importers/yum/listener.py

```diff
@@ -78,7 +79,11 @@ def download_succeeded(self, report):
         if isinstance(model, (models.RPM, models.SRPM)):
             self.metadata_files.add_repodata(model)
         # init unit, which is idempotent
-        unit = self.sync_conduit.init_unit(model.TYPE, model.unit_key, model.metadata, model.relative_path)
+        unit = self.sync_conduit.init_unit(model.TYPE, model.unit_key, model.metadata,
+                                           model.relative_path)
```

Comment:
I believe that Jenkins has been passing on the 2.6 branch, so it should be OK. Likely this is because the 2.6 may not have had the strict PEP-8 checks yet. These issues are likely already fixed on master since the flake8 checks are enforced there and those builds also pass.